winpr/libwinpr/sspi/Schannel/schannel_openssl.c

```diff
@@ -237,7 +235,7 @@ SECURITY_STATUS schannel_openssl_client_process_tokens(SCHANNEL_OPENSSL* context
 			if (!pBuffer)
 				return SEC_E_INVALID_TOKEN;
 
-			status = BIO_write(context->bioRead, pBuffer->pvBuffer, pBuffer->cbBuffer);
+			BIO_write(context->bioRead, pBuffer->pvBuffer, pBuffer->cbBuffer);
```

Comment:
the SChannel SSPI module is really a WIP... what often happens with WIP code is that a lot of status codes are unchecked. By removing all of them because of that, they will need to be re-added later when the work on the WIP code continues. Obtaining the status code, even if unused, is a reminder that there is something to be checked there, like a todo.